subt_ros/src/SubtRosRelay.cc

```diff
@@ -222,6 +229,26 @@ SubtRosRelay::SubtRosRelay()
     this->rosnode->advertise<std_msgs::Int32>("score", 1000);
   this->rosCompetitionClockPub =
     this->rosnode->advertise<subt_ros::CompetitionClock>("run_clock", 1000);
+
+  // Setup a ros bag recorder.
+  rosbag::RecorderOptions recorderOptions;
+  recorderOptions.append_date=false;
+  recorderOptions.split=true;
+  recorderOptions.max_splits=1;
```

Comment:
Before it was 2 :)

Reply:
Yes, please don't make the logs smaller, thanks.

Reply:
The previous implementation was incorrect. According to the [API](https://github.com/osrf/subt/wiki/api#custom-data-logging), up to 2 GB across 2 log files should be recorded. This equates to 1 split, which allows 2 log files.

This PR will produce up two 2 bag files, each with a maximum of 1GB.